code/obj/flock/structure/collector.dm

```diff
@@ -35,7 +35,8 @@
 		icon_state = "collectoron"
 	else
 		icon_state = "collector"
-	src.poweruse = ((length(connectedto) * 5) / -1) //(5 power per tile)
+	//TODO: Rebalance this when we do the proper compute strucutres and computers pass
```

Comment:
?

Reply:
we ended up balancing around this instead, so I'll just remove comment

Reply:
https://github.com/stonepillars/goon-flock/pull/333